File: test_archives/py_functions/batch2/function4.py
def function4 (num1, num2, num3, string1, list1): 
  '''
  Function4 takes 5 inputs: two numbers, 1 string, and a list.
  It adds together the numbers, concatenates the string,
  and sums all the elements of the list.
  It returns the final sum.
  '''
  return num1 + num2 + num3 + sum(list1) + len(string1)

File: test_archives/cpp_functions/batch2/function4.cpp
int function4(int num1, int num2, int num3, string string1, vector<int> list1) {
  int result = num1 + num2 + num3;
  int list1Sum = 0;
  for (int i = 0; i < list1.size(); i++) {
    list1Sum += list1[i];
  }
  result += list1Sum + string1.length();
  return result;
}